samples/nrf9160/nrf_cloud_rest_device_message/prj.conf

```diff
@@ -1,6 +1,6 @@
 # Sample
 CONFIG_REST_DEVICE_MESSAGE_DO_JITP=n
-CONFIG_NRF_CLOUD_REST_DEVICE_MESSAGE_SAMPLE_LOG_LEVEL_INF=y
+CONFIG_NRF_CLOUD_REST_DEVICE_MESSAGE_SAMPLE_LOG_LEVEL_DBG=y
```

Comment:
I guess I should clean this up now, rather than waiting as I mentioned in the commit message...
```
NOTE: changes to prj.conf are mostly for testing;
reduce to minimum before PR merged.